constants/wram_constants.asm

```diff
@@ -56,7 +56,7 @@ DEF NUM_FRAMES EQU const_value
 ; wTextboxFlags::
 	const_def
 	const FAST_TEXT_DELAY_F ; 0
-	const NO_TEXT_DELAY_F   ; 1
+	const TEXT_DELAY_F      ; 1
```

Comment:
When this ``wTextboxFlags`` bit is set, ``PrintLetterDelay``, prints letter delay, further checking ``FAST_TEXT_DELAY_F``. 
When this ``wTextboxFlags`` bit is not set, ``PrintLetterDelay`` returns without applying a letter delay.

``NO_TEXT_DELAY_F`` is misleading because it reflects the behavior when the bit is not set, rather than set.

Reply:
Makes sense. Thanks!